app/src/main/java/com/ghomebyrw/gworker/clients/JobClient.java

```diff
@@ -52,4 +52,13 @@ public void updateJobPrice(String jobId, @Body Price jobPrice, Callback<Job> htt
 
         // TODO - update once API is deployed to UAT
     }
+
+    public void logIn(String userName, String password, Callback<Boolean> httpHandler) {
+        Retrofit retrofit = new Retrofit.Builder()
+                .baseUrl(BASE_URL)
+                .addConverterFactory(GsonConverterFactory.create(getDeserializer()))
+                .build();
+        TooltimeAPI service = retrofit.create(TooltimeAPI.class);
+        service.logIn().enqueue(httpHandler);
```

Comment:
I know it will be updated anyway when the api is ready, but we'll definitely need to pass `userName` and `password` to `TooltimeAPI.logIn`.